ledger/migration/src/lib.rs

```diff
@@ -80,6 +80,26 @@ pub fn migrate(ledger_db_path: impl AsRef<Path>, logger: &Logger) {
                 );
                 db_txn.commit().expect("Failed committing transaction");
             }
+            // Version 20220222 came after 20200707 introduced minting.
+            Err(MetadataStoreError::VersionIncompatible(20200707, _)) => {
+                log::info!(
+                    logger,
+                    "Ledger db migrating from version 20200707 to 20220222..."
+                );
+                MintConfigStore::create(&env).expect("Failed creating MintConfigStore");
+
+                let mut db_txn = env.begin_rw_txn().expect("Failed starting rw transaction");
+                metadata_store
+                    .set_version_to_latest(&mut db_txn)
```

Comment:
Sorry for my ignorance, but is there a benefit of setting to latest vs specifying 20220222 here?

Reply:
Good catch, that was actually a potential bug!
We need to be explicit with the version we set to (and not just set to latest) so that incremental migration steps would work.